python-enum/cardinals.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum, auto
+
+# class CardinalDirection(enum.Enum):
```

Comment:
:wrench: **Technical Recommendation**

Let's be consistent and use the same import style throughout:

```suggestion
# class CardinalDirection(Enum):
```